Samples/OpenQasmReader.Tests/Validate/Teleport.qs

```diff
@@ -0,0 +1,34 @@
+﻿namespace Microsoft.Quantum.Samples.OpenQasmReader.Tests.Validate
+{
+    open Microsoft.Quantum.Primitive;
+    open Microsoft.Quantum.Canon;
+    open Microsoft.Quantum.Extensions.Math;
+
+    operation TeleportTest():(Result[])
+    {
+        body
+        {
+            mutable c = new Result[3];
+            using(q = Qubit[3]){
+				Rx(0.7,q[0]); 
+				Ry(0.8,q[0]); 
+				Rz(0.9,q[0]);
+				H(q[1]);
+				CNOT(q[1],q[2]);
+				CNOT(q[0],q[1]);
+				H(q[0]);
+				set c[0] = M(q[0]);
+				set c[1] = M(q[1]);
+				if(c[0]==1){
```

Comment:
this is not going to work, because c[] is Result so it can't be compared to Int...

Reply:
Maybe we'll need to convert all measurements to Int?

Reply:
there is [ResultAsInt](https://docs.microsoft.com/en-us/qsharp/api/canon/microsoft.quantum.canon.resultasint?view=qsharp-preview), although that expects an array of Results.